stdlib.go

```diff
@@ -135,16 +143,17 @@ const (
 )
 
 var (
-	logRegexp = regexp.MustCompile(logRegexpDate + logRegexpTime + logRegexpFile + logRegexpMsg)
+	StdlibLogRegexpDateTimeFileMsg = regexp.MustCompile(logRegexpDate + logRegexpTime + logRegexpFile + logRegexpMsg)
+	StdlibLogRegexpDateTimeMsg     = regexp.MustCompile(logRegexpDate + logRegexpTime + logRegexpMsg)
 )
 
-func subexps(line []byte) map[string]string {
-	m := logRegexp.FindSubmatch(line)
-	if len(m) < len(logRegexp.SubexpNames()) {
+func (a StdlibAdapter) subexps(line []byte) map[string]string {
+	m := a.logRegexp.FindSubmatch(line)
```

Comment:
This will panic if `a.logRegexp` is nil, so let's prevent that, somehow: either by checking it explicitly here, or by ensuring that the Regexp option will reject nil values.

Reply:
I add a nil check on the option func